Package.swift

```diff
@@ -0,0 +1,24 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "Autofill",
+    products: [
+        .library(
+            name: "Autofill",
+            targets: ["Autofill"]),
+    ],
+    dependencies: [
+    ],
+    targets: [
+        .target(
+            name: "Autofill",
+            dependencies: [],
+            resources: [
+                .process("dist")
```

Comment:
Wouldn't the "copy" instead of "process" rule be more appropriate?

Reply:
I guess I wasn't sure if we might want some of the processing later; but yeah I think for now that's fine. Will change.

Reply:
Without .process here it seems we always retain the 'dist' in the file structure which is a little ugly, the alternative is copy each file which is even worse.

I was hoping for .copy('dist/*') or similar syntax that wouldn't keep dist/ in the output.

Reply:
Landed on using 'assets' as a dir prefix when speaking with @miasma13 over MM, I think this is the best compromise here.